cdap-common/src/test/java/io/cdap/cdap/common/RepositoryNotFoundExceptionTest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.common;
+
+import io.cdap.cdap.proto.id.NamespaceId;
+import org.junit.Assert;
+import org.junit.Test;
+
+/** Tests fpr RepositoryNotFoundException */
+public class RepositoryNotFoundExceptionTest {
+  @Test
+  public void testRepositoryNotFoundExceptionMessage() {
+    RepositoryNotFoundException exception = new RepositoryNotFoundException(NamespaceId.DEFAULT);
+
+    Assert.assertEquals(
+        exception.getMessage(),
+        String.format("There is no repository configuration for namespace %s. "
+            + "Please configure it in Namespace Admin page.", NamespaceId.DEFAULT.getNamespace()));
```

Comment:
Please don't use String.format over here. It does not give much, but prevents from seeing exact error message served to user

Reply:
Also I believe in `org.junit.Assert#assertEquals(java.lang.Object, java.lang.Object)` expected goes first in params

Reply:
updated